scripts/lib/zulip_tools.py

```diff
@@ -264,3 +252,18 @@ def generate_sha1sum_emoji(zulip_path):
     sha.update(emoji_datasource_version)
 
     return sha.hexdigest()
+
+def may_be_perform_purging(dirs_to_purge, dirs_to_keep, dir_type, dry_run):
+    # type: (Set(Text), Set(Text), Text, bool) -> None
+    if dry_run:
+        print("Performing a dry run...")
+    else:
+        print("Cleaning unused %ss..." % (dir_type,))
```

Comment:
I have used an extra `s` in the string to print `emoji caches` instead of `emoji cache`. Feel free to remove/modify it if it doesn't suit your taste.